### PR TITLE
Convert eeoc dofile

### PR DESCRIPTION
**Pull request by jhirsch1**

### CODE REVIEW

File: code/clean/merge_all.do [hunk 1]
```diff
@@ -23,39 +23,39 @@ Clean EEOC court case data from .py files to match state files
```
Comment (amarchenko26) [inline code/clean/merge_all.do:23]: Update or delete this comment, no longer accurate re: .py files

File: code/clean/merge_all.do [hunk 2]
```diff
@@ -23,39 +23,39 @@ Clean EEOC court case data from .py files to match state files
...
-replace basis_clean = "Disability" if regexm(basis, "^ADA") //Americans with Disabilities
-replace basis_clean = "Age" if regexm(basis, "^ADEA") //Age Discrimination in Employment
-replace basis_clean = "Retaliation" if regexm(basis, "^Title VII / Retaliation") | regexm(basis, "^EPA / Retaliation")
-replace basis_clean = "Other" if regexm(basis, "^Title VII / Other") | regexm(basis, "^GINA") | basis == "" // if it's missing
-
-g win = 1 if missing_relief == 0
-replace win = 0 if missing_relief == 1 // no probable cause if relief is missing
-
-g court = 1 
+//
```
Comment (amarchenko26) [inline code/clean/merge_all.do:26]: Delete this code instead of commenting it out

File: code/clean/clean_eeoc.do [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+ren casename resp_org
+ren civilactionnumber Civil_Action_Number
+ren court court_name
+ren courtfilingdate Court_Filing_Date_temp
+ren resolutiondate Resolution_Date_temp
+ren relief Relief_temp
+ren allegations basis
+
+ren *, lower
```
Comment (amarchenko26) [inline code/clean/clean_eeoc.do:22]: Do we still need to ren *, lower or can we just rename variables directly to lower case in lines 14-20?

File: code/clean/clean_eeoc.do [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+// Remove the dollar sign and commas from the 'Relief' column
+replace relief_temp = regexreplaceall(relief_temp, "[$,]", "")
+
+// Create a new column to indicate missing relief
+gen missing_relief = 1 if missing(relief_temp)
+replace missing_relief = 0 if missing_relief != 1
+//
+// // Convert to numeric, coercing errors to NaN, and fill NaNs with 0
+replace relief_temp = "0" if missing(relief_temp)
```
Comment (amarchenko26) [inline code/clean/clean_eeoc.do:63]: I just checked the README, and our protocol is now to not make relief == 0 when plaintiff lost. When plaintiff lost, relief == .

Pls remove this

File: code/clean/clean_eeoc.do [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+destring relief_temp, gen(relief)
+
+drop relief_temp court_filing_date_temp resolution_date_temp
+
+
+
+/*******************************************************************************
+Clean EEOC court case data from .py files to match state files
```
Comment (amarchenko26) [inline code/clean/clean_eeoc.do:71]: Update comment

File: code/clean/clean_eeoc.do [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+gen court_res_date = date(resolution_date_temp, "MDY"), after(resolution_date_temp)
+
+// Calculate the duration in days between filing and resolution
+gen duration = court_res_date - court_file_date
+
+// Remove the dollar sign and commas from the 'Relief' column
+replace relief_temp = regexreplaceall(relief_temp, "[$,]", "")
+
+// Create a new column to indicate missing relief
+gen missing_relief = 1 if missing(relief_temp)
```
Comment (amarchenko26) [inline code/clean/clean_eeoc.do:59]: What is relief_temp? In 55-63 I feel like we're creating a bunch of vars unnecessarily? Why can't we just: 

g missing_relief = (relief ==0)

is that going to do everything in those 10 lines?

**Review by amarchenko26 — COMMENTED** [review]
Left some comments, pls make changes and then i'll quickly check and merge in this branch. Then, let's move on to overlap cases to make some progress by tomorrow.

File: code/clean/clean_eeoc.do (hunk 6 above)
Comment (jhirsch1) [reply code/clean/clean_eeoc.do:59]: One other thing these lines do is convert the relief values to numbers from strings. I did remove the extra temporary value though and clean it up.

**Review by amarchenko26 — APPROVED** [review]
Looks good thank you!

**Pull request merged**